multimodal/src/autogluon/multimodal/models/mmocr_text_detection.py

```diff
@@ -1,33 +1,12 @@
 import logging
-import warnings
 from typing import Optional
 
-import torch
-from mim.commands.download import download
+from mmcv.parallel import scatter
```

Comment:
I think it is better to put these import inside `try except` block, something like this (https://github.com/awslabs/autogluon/blob/master/multimodal/src/autogluon/multimodal/models/mmdet_image.py#L17-L22). It may remove the error in CI.